seatunnel-e2e/seatunnel-connector-v2-e2e/connector-prometheus-e2e/src/test/java/org/apache/seatunnel/e2e/connector/prometheus/PrometheusIT.java

```diff
@@ -0,0 +1,98 @@
+ */
+package org.apache.seatunnel.e2e.connector.prometheus;
+
+import org.apache.seatunnel.e2e.common.TestResource;
+import org.apache.seatunnel.e2e.common.TestSuiteBase;
+import org.apache.seatunnel.e2e.common.container.EngineType;
+import org.apache.seatunnel.e2e.common.container.TestContainer;
+import org.apache.seatunnel.e2e.common.junit.DisabledOnContainer;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.TestTemplate;
+import org.testcontainers.containers.Container;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.containers.wait.strategy.HostPortWaitStrategy;
+import org.testcontainers.lifecycle.Startables;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.DockerLoggerFactory;
+
+import com.google.common.collect.Lists;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.util.stream.Stream;
+
+@Slf4j
+public class PrometheusIT extends TestSuiteBase implements TestResource {
+
+    private static final String IMAGE = "bitnami/prometheus:2.53.0";
+
+    private GenericContainer<?> prometheusContainer;
+
+    private static final String HOST = "prometheus-host";
+
+    @BeforeAll
+    @Override
+    public void startUp() {
+        this.prometheusContainer =
+                new GenericContainer<>(DockerImageName.parse(IMAGE))
+                        .withNetwork(NETWORK)
+                        .withNetworkAliases(HOST)
+                        .withEnv("TZ", "Asia/Shanghai")
+                        .withExposedPorts(9090)
+                        .withCommand(
+                                "--config.file=/opt/bitnami/prometheus/conf/prometheus.yml",
+                                "--web.enable-remote-write-receiver")
+                        .withLogConsumer(new Slf4jLogConsumer(DockerLoggerFactory.getLogger(IMAGE)))
+                        .waitingFor(
+                                new HostPortWaitStrategy()
+                                        .withStartupTimeout(Duration.ofMinutes(2)));
+        prometheusContainer.setPortBindings(Lists.newArrayList(String.format("%s:9090", "9090")));
+        Startables.deepStart(Stream.of(prometheusContainer)).join();
+        log.info("Mongodb container started");
+    }
+
+    @AfterAll
+    @Override
+    public void tearDown() {
+        if (prometheusContainer != null) {
+            prometheusContainer.stop();
+        }
+    }
+
+    @TestTemplate
+    @DisabledOnContainer(
+            value = {},
+            type = {EngineType.SPARK},
+            disabledReason = "Currently protobuf version is conflict")
+    public void testPrometheusSinkAndSource(TestContainer container)
+            throws IOException, InterruptedException {
+        //
+        // http prometheus
```

Comment:
`disabledReason` Is there any related info, want to take a look. One more small detail, extra '//'